New blueprint: use runPHP to add a woocommerce product

mujuonly opened the pull request:
The goal:

Create a product with additional metadata using `runPHP` option with `wp_insert_post`
Use `phpExtensionBundles` - `kitchen-sink` to generate XML file
Redirect to the product feed generating page using `landingPage`

adamziel commented:
Thank you for contributing this, @mujuonly! It looks good, I'd just move it from a `runphp` directory to something more descriptive, such as `woocommerce-product-feed`. Don't worry about the failing CI checks, that's something to fix on my end.

adamziel committed:
CI automations improvement

Switch to pull_request_target in preview-comment.yml.

Compare the updated files to merge-base trunk instead of trunk in
validate_pr.yml.

adamziel commented:
Oops wrong command, let me revert this and restore the original change

adamziel commented:
Done in https://github.com/adamziel/blueprints/commit/f5f2cef7987585ba89e8d143d4cd7455e2fc211f, I also renamed the directory in https://github.com/adamziel/blueprints/commit/f5cccb208ea2dd93c769c19c2b2fd6af606f5dac
